.swiftpm/xcode/xcshareddata/xcschemes/Confidence-Package.xcscheme

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am confused if this code should be checked into the repo (I think yes)

Reply:
i think we shouldn't commit these files.

Reply:
Removed 👍